lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -237,15 +237,12 @@ const (
 
 func (c *Client) readClientSize() error {
 	for {
-		msg, err := c.cfg.Conn.ReadMessage()
+		s, err := c.cfg.Conn.ReadClientScreenSpec()
 		if err != nil {
-			return trace.Wrap(err)
-		}
-		s, ok := msg.(tdp.ClientScreenSpec)
-		if !ok {
-			c.cfg.Log.Debugf("Expected ClientScreenSpec message, got %T", msg)
+			c.cfg.Log.Debug(err)
```

Comment:
Include some more context in the error, otherwise the debug logs are going to show an error that's hard to troubleshoot.

Reply:
4cb06925c7d703ecc21f5b874c661eeb7d0f43fc